mongorestore/mongorestore.go

```diff
@@ -35,10 +35,12 @@ type MongoRestore struct {
 
 	TargetDirectory string
 
+	SkipUsersAndRoles bool
```

Comment:
Same here... InputOptions already has a flag related the restoring db users and roles.